src/enums.py

```diff
@@ -2,5 +2,5 @@
 
 
 class SearchBehavior(Enum):
```

Comment:
This is not a change that needs to be done on this PR, just a note for the future - I think that this class can be moved into `constants.py` and `enums.py` can be dropped

Reply:
agreed